### ISSUE TITLE
Poco F6 - Request parameter error (10000)

### ISSUE THREAD

**Issue by BgenkBlackisme** [MlgmXyysd/Xiaomi-HyperOS-BootLoader-Bypass#180]
Poco F6 - Request parameter error (10000)

![error 10000](https://github.com/user-attachments/assets/166e8d77-e80c-42e4-b914-0c096e91a1f4)

**Comment by BgenkBlackisme** [comment]
my hyperos
![Screenshot_2024-09-22-13-04-12-806_com android settings](https://github.com/user-attachments/assets/0784fd6f-d2c1-427c-89cd-a87eda89780d)

**Comment by fillahfirdausyah** [comment]
same issue here

**Comment by AmihayHassan** [comment]
same here, Poco F6 running HyperOS 1.0.7.0 UNPMIXM

added `var_dump($args);` in line 333

this is the output

![image](https://github.com/user-attachments/assets/88cbbff7-339d-4fd0-a1bc-e5a3e9a5947b)

**Comment by BgenkBlackisme** [comment]
I will try tomorrow, thanks for your respond bro

**Comment by BgenkBlackisme** [comment]
Still Cant Solved

**Comment by AmihayHassan** [comment]
Thank for the quick reply @MlgmXyysd
Is there anything we can do to help?

**Comment by AmihayHassan** [comment]
> Still Cant Solved

Thank for the quick reply, is there anything we can do to help?

**Comment by BgenkBlackisme** [comment]
after quota limit, i got account error, wtf

**Comment by BlazingRiser** [comment]
Go in pull requests, there is one open post, inside there is link to 123pan, download your device specific settings apk from there that should fix this issue. Although turbo 3 had issue with error 30003.

**Comment by AmihayHassan** [comment]
> Go in pull requests, there is one open post, inside there is link to 123pan, download your device specific settings apk from there that should fix this issue. Although turbo 3 had issue with error 30003.

Not speaking Chinese but somehow managed to get there, anyway, I couldn't download it, and other versions (from apkmirror) are failing to install

**Comment by JarJarSphynx** [comment]
> Go in pull requests, there is one open post, inside there is link to 123pan, download your device specific settings apk from there that should fix this issue. Although turbo 3 had issue with error 30003.

except that there is no apk in this list for the F6, only the turbo3.

**Comment by AmihayHassan** [comment]
> > Go in pull requests, there is one open post, inside there is link to 123pan, download your device specific settings apk from there that should fix this issue. Although turbo 3 had issue with error 30003.
> 
> except that there is no apk in this list for the F6, only the turbo3. 

I think this is basically the same device..

**Comment by betamaster-codes** [comment]
We would have a fix for Poco F6 5G soon for sure - kudos to the fantastic project
It is just a matter of time

**Comment by marufmoinuddin** [comment]
> We would have a fix for Poco F6 5G soon for sure - kudos to the fantastic project It is just a matter of time

can we expect a fix for redmi note 13 pro 5g too?

**Comment by AmihayHassan** [comment]
> We would have a fix for Poco F6 5G soon for sure - kudos to the fantastic project
> It is just a matter of time

I wouldn't count on it, the last release was ~11m ago

**Comment by JarJarSphynx** [comment]
> > > Go in pull requests, there is one open post, inside there is link to 123pan, download your device specific settings apk from there that should fix this issue. Although turbo 3 had issue with error 30003.
> > 
> > 
> > except that there is no apk in this list for the F6, only the turbo3.
> 
> I think this is basically the same device..

No it's literally not. you can't install the turbo 3 setting apk on the f6 in file manager, adb or any way, besides if you try with a mi account on the f6 it will keep prompting you to try again after two weeks regardless of whether your account region was set to global or your country so you will never be able to unlock it. tl;dr: everyone including the devs who made this fix pretty much left the f6 for dead here.

**Comment by BlazingRiser** [comment]
> > > > Go in pull requests, there is one open post, inside there is link to 123pan, download your device specific settings apk from there that should fix this issue. Although turbo 3 had issue with error 30003.
> > > 
> > > 
> > > except that there is no apk in this list for the F6, only the turbo3.
> > 
> > 
> > I think this is basically the same device..
> 
> No it's literally not. you can't install the turbo 3 setting apk on the f6 in file manager, adb or any way, besides if you try with a mi account on the f6 it will keep prompting you to try again after two weeks regardless of whether your account region was set to global or your country so you will never be able to unlock it. tl;dr: everyone including the devs who made this fix pretty much left the f6 for dead here.

It's the same device, around August Xiaomi blocked the bypass for three devices shipping hyperos out of the box, before August the bypass used to work for f6, you can still unlock via official/or paid service. You can check devices which are blocked in [status.md](https://github.com/MlgmXyysd/Xiaomi-HyperOS-BootLoader-Bypass/blob/master/status.md)

**Comment by betamaster-codes** [comment]
Basically has moved of - to different ecosystem
No more "Xiaomi" in all my connection,, extended connections and professional connection

I have made people more than 400+ users, leave xiaomi devices who all once enjoyed xiaomi, as I configured their devices the way they wanted it to work.

Now a new ecosystem - No more xiaomi for all my connections

Loss for xiaomi - nothing for us to loose.
@xiaomi, @poco, @redmi and all their devices for making our lives difficult 

'm educating more and more poeple to leave Xiaomi and their shit policies - even after paying for the device and owning it - no company including xiaomi has any right what we as user do with our devices - whether unlook, root etc

But it seems they want to control devices ownership, even after user bought the device - paid the money 
This is simply unfair trade practice,
Milk seller cannot dictate what we do with milk once we buy ?
Xiaomi as mobile seller cannot dictate, what we do with our devices, do they then answer is "middle finger" to xiaomi

Thank you guys for this fantastic project!
Cheers
Bm

**Comment by JarJarSphynx** [comment]
> > > > > Go in pull requests, there is one open post, inside there is link to 123pan, download your device specific settings apk from there that should fix this issue. Although turbo 3 had issue with error 30003.
> > > > 
> > > > 
> > > > except that there is no apk in this list for the F6, only the turbo3.
> > > 
> > > 
> > > I think this is basically the same device..
> > 
> > 
> > No it's literally not. you can't install the turbo 3 setting apk on the f6 in file manager, adb or any way, besides if you try with a mi account on the f6 it will keep prompting you to try again after two weeks regardless of whether your account region was set to global or your country so you will never be able to unlock it. tl;dr: everyone including the devs who made this fix pretty much left the f6 for dead here.
> 
> It's the same device, around August Xiaomi blocked the bypass for three devices shipping hyperos out of the box, before August the bypass used to work for f6, you can still unlock via official/or paid service. You can check devices which are blocked in [status.md](https://github.com/MlgmXyysd/Xiaomi-HyperOS-BootLoader-Bypass/blob/master/status.md)

Nope, same hardware maybe, not the same target firmware. I tried the turbo 3 apk, tried to install by adb and even edited the apk myself and tried doing workarounds, and finally when it's installed it force closes, it doesn't even have the same gui/icon btw.
as for the f6 you can never unlock it by the official method as per my previous reply, and btw even paid services like ncunblock don't have a service for the f6 on their website.
tl;dr if anyone bought the f6 past august and expects to unlock it, sell it. don't bother trying to unlock it. it's a lost case.

**Comment by JarJarSphynx** [comment]
> Basically has moved of - to different ecosystem No more "Xiaomi" in all my connection,, extended connections and professional connection
> 
> I have made people more than 400+ users, leave xiaomi devices who all once enjoyed xiaomi, as I configured their devices the way they wanted it to work.
> 
> Now a new ecosystem - No more xiaomi for all my connections
> 
> Loss for xiaomi - nothing for us to loose. @XiaoMi, @Poco, @redmi and all their devices for making our lives difficult
> 
> 'm educating more and more poeple to leave Xiaomi and their shit policies - even after paying for the device and owning it - no company including xiaomi has any right what we as user do with our devices - whether unlook, root etc
> 
> But it seems they want to control devices ownership, even after user bought the device - paid the money This is simply unfair trade practice, Milk seller cannot dictate what we do with milk once we buy ? Xiaomi as mobile seller cannot dictate, what we do with our devices, do they then answer is "middle finger" to xiaomi
> 
> Thank you guys for this fantastic project! Cheers Bm

good luck. good move. the only reason people are buying their smartphones at this point is a good chip for affordable price. the tradeoff is that everything else = low quality whether software or hardware.

**Comment by Mostwest** [comment]
Nice, I just bought my Poco F6 3 days ago...
I'm not even able to use the regular activation, this is what happens at 0:00 China time

The Xiaomi Community app can't even load the page because the servers are oversaturated by requests

I hoped to find a way but...

![1732661578631](https://github.com/user-attachments/assets/5dd57528-d519-4a3a-8841-f38bc85d01dd)

**Comment by vpolinov** [comment]
> No more "Xiaomi"

support 100%

**Comment by cute-omega** [comment]
here mentioned error code `10000` https://github.com/xbn2002/Xiaomi-HyperOS-BootLoader-Bypass/tree/patch-1
and for anyone who need to downgrade setting app, here may be something you need https://github.com/code4fun1024/setting-app-for-hyperos

**Comment by JarJarSphynx** [comment]
> here mentioned error code `10000` https://github.com/xbn2002/Xiaomi-HyperOS-BootLoader-Bypass/tree/patch-1 and for anyone who need to downgrade setting app, here may be something you need https://github.com/code4fun1024/setting-app-for-hyperos

No it doesn't work. xiaomi blocked downgrading the setting app, there's literally no way you can install any of these apks.
I've been saying this since 2024. absolutely no one can downgrade his settings app as of this moment, regardless of any settings.apk you try to install (even the one that alligns with your rom/device)

![xiaomi](https://github.com/user-attachments/assets/6955e18c-c40c-47cf-afe8-c40594e03372)

and on top of that, they blocked redmi turbo 3 from unlocking without an account, and you can't unlock with an account no matter how long your account is since it's creation because the community app will ask you to try again after two weeks!

just never purchase anything from xiaomi again, genuine advice.

> > > Go in pull requests, there is one open post, inside there is link to 123pan, download your device specific settings apk from there that should fix this issue. Although turbo 3 had issue with error 30003.
> > 
> > 
> > except that there is no apk in this list for the F6, only the turbo3.
> 
> I think this is basically the same device..

**Comment by Mostwest** [comment]
> here mentioned error code `10000` https://github.com/xbn2002/Xiaomi-HyperOS-BootLoader-Bypass/tree/patch-1 and for anyone who need to downgrade setting app, here may be something you need https://github.com/code4fun1024/setting-app-for-hyperos

Yeah this stopped working around september for Poco F6/redmi Turbo updated devices.

Downgrading doesn't work. 
You need an account with permission to unlocked it.
I went for the paid way but the guy who always unlock phones for many wasn't able to do it for my phone 2 weeks ago, he did refund me.

**Comment by JarJarSphynx** [comment]
> > here mentioned error code `10000` https://github.com/xbn2002/Xiaomi-HyperOS-BootLoader-Bypass/tree/patch-1 and for anyone who need to downgrade setting app, here may be something you need https://github.com/code4fun1024/setting-app-for-hyperos
> 
> Yeah this stopped working around september for Poco F6/redmi Turbo updated devices.
> 
> Downgrading doesn't work. You need an account with permission to unlocked it. I went for the paid way but the guy who always unlock phones for many wasn't able to do it for my phone 2 weeks ago, he did refund me.

yeah well as I've already mentioned like 5 times before not even their official app works and it'll just ask you to try again after two weeks even if you apply on time (00:00 in mainland) with only sim data activated and global region

![Image](https://github.com/user-attachments/assets/c83c53c0-90c0-450e-aeed-b32ffc90a6cc)

basically 90% of poco f6 users are completely stuck. only those who bought it when it first came out and unlocked it within 3 or 4 months got lucky.
